site/content/entry/yuzu-progress-report-jun-2021/index.md

```diff
@@ -0,0 +1,307 @@
+The only exception is Intel with their integrated HD Graphics and UHD Graphics series.
+
+[In the past,](https://yuzu-emu.org/entry/yuzu-progress-report-feb-2021/) epicboy implemented a way to accelerate ASTC texture decoding via the use of `Compute Shaders`,
+improving decoding performance considerably thanks to taking advantage of the great computing power of modern GPUs.
+
+The issue is that in some games, for example `Astral Chain`, a synchronization issue caused yuzu to try to access a texture before its decoding was finished, resulting in 
+driver panics and application crashes. 
+[Implementing various optimizations and enhancements to the GPU accelerated decoder](https://github.com/yuzu-emu/yuzu/pull/6496) solved those crashes, even on a simple GT 
+730, a card 41.5x times weaker than an RTX 2080 SUPER in compute performance.
+
+After Project Hades is finished, there are plans to implement performance optimizations on the ASTC GPU accelerated decoder.
+
+Previous to this work, [a toggle to disable the GPU accelerated decoder](https://github.com/yuzu-emu/yuzu/pull/6464) was added for debugging purposes.
+It’s no longer needed, but if anyone is curious about how much of a difference decoding with the CPU makes, the option is in `Emulation > Configure… > Graphics > 
+Accelerate ASTC texture decoding`.
+
+By [avoid creating image views for blits for different bytes per block, ](https://github.com/yuzu-emu/yuzu/pull/6469) Rodrigo solved crashes experienced by Unreal Engine 4 
+games on Vulkan.
+
+Speaking about blits, [vonchenplus](https://github.com/vonchenplus) found out our previous way of handling out of bounds texture blits wasn’t accurate enough, causing 
+rendering glitches in games like `DRAGON QUEST III The Seeds of Salvation`.
+[Adding an offset to the source texture address](https://github.com/yuzu-emu/yuzu/pull/6531) puts this bug to rest.
+
+{{< single-title-imgs
+    "No more acne! (DRAGON QUEST III: The Seeds of Salvation)"
+    "./blitbug.png"
+    "./blitfix.png"
+    >}}
+
+Some games running in Vulkan like `Super Smash Bros. Ultimate` or `A Hat in Time` can cause loops decoding textures, resulting in `Out of Bounds` access on an array, 
+potentially leading to a crash, and breaking our Flatpak support.
+[As preemptive work](https://github.com/yuzu-emu/yuzu/pull/6410), [toastUnlimited](https://github.com/lat9nq) added a `break` safeguard, and now Flatpaks work as intended.
+Having less crashes is always better, right?
+
+## General changes and bugfixes
+
+`discord-rpc`, the submodule in charge of handling Discord’s `Rich Presence` “Now Playing” feature, [was updated](https://github.com/yuzu-emu/yuzu/pull/6484) by 
+[Vortex](https://github.com/CaptV0rt3x) to the last version before it was deprecated in favour of a closed-source alternative named `GameSDK` which of course we can’t use.
+Since discord-rpc still is compatible with Rich Presence, this update should provide more stable reporting in your Discord Status.
```

Comment:
```suggestion
Since `discord-rpc` still is compatible with `Rich Presence`, this update should provide more stable reporting in your Discord Status.
```